draftlogs/7092_fix.md

```diff
@@ -0,0 +1,2 @@
+ - Fix satellite-streets with map traces [[#7092](https://github.com/plotly/plotly.js/pull/7092)]
+ 
```

Comment:
Thanks for the PR.
Let's remove this file and add this PR to the list of maplibre PRs in 7060_add.md.